strings/languages/af/strings.json

```diff
@@ -84,7 +84,8 @@
       "showBatteryAndTime": "Show battery and time",
       "showProgressPercentage": "Show progress percentage",
       "swipeGestures": "Swipe left or right to navigate between chapters",
-      "allowTextSelection": "Allow text selection"
+      "allowTextSelection": "Allow text selection",
+      "showChapterName": "Show chapter name"
```

Comment:
You just need to add the string to `strings/languages/en/strings.json`, Crowdin would automatically add it to rest of the files.